draft-ietf-scitt-architecture.md

```diff
@@ -192,6 +191,12 @@ Auditor:
 
 : an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service.
 
+Client:
```

Comment:
I believe this role is outside the scope of SCITT and hence this PR should be abandoned!